### DISCUSSION TITLE
词库需要更新

### DISCUSSION THREAD

**Discussion by ADSR1042** [ADSR1042/ZJU_VocabBoost#10] [Announcements]
2023年秋起大英教材替换了部分阅读及配套词汇，相应词库还未更新。
本人精力所限，暂无力校对更新。希望有前辈能帮忙，直接发PR即可。
感谢!

**Comment by Ayanami-WU** [comment]
其实我才发现原来的词库挺好的
因为我们老师似乎还是在用原来的词库
